src/single-spa-layout.js

```diff
@@ -1,3 +1,4 @@
 export { constructRoutes } from "./isomorphic/constructRoutes.js";
 export { constructApplications } from "./browser/constructApplications.js";
 export { constructLayoutEngine } from "./browser/constructLayoutEngine";
+export { matchRoute } from "./isomorphic/matchRoute.js";
```

Comment:
I wasn't sure if this should be exported from single-spa-layout.js or single-spa-layout-server.js. Since it's isomorphic, I think it makes sense to have it in single-spa-layout.js. This also will match the documentation you linked to.